src/main/java/io/github/opencubicchunks/cubicchunks/cubicgen/customcubic/structure/IFlexHandlerStructureGenerator.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  This file is part of Cubic World Generation, licensed under the MIT License (MIT).
+ *
+ *  Copyright (c) 2015-2020 contributors
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package io.github.opencubicchunks.cubicchunks.cubicgen.customcubic.structure;
+
+import io.github.opencubicchunks.cubicchunks.api.util.CubePos;
+import io.github.opencubicchunks.cubicchunks.api.worldgen.CubePrimer;
+import io.github.opencubicchunks.cubicchunks.api.worldgen.structure.ICubicStructureGenerator;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+public interface IFlexHandlerStructureGenerator extends ICubicStructureGenerator {
```

Comment:
what does that name mean?

Reply:
i thought about giving it some suitable name. 

Flex abbreaviation for Flexible, didnt even realise when i typed it. So a Flexible handler generator, as it has a method returning the handler.

if you have a better name please tell me